prime-dotnet-webapi/Migrations/20240318232909_EmailTemplateUpdate.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace Prime.Migrations
+{
+    public partial class EmailTemplateUpdate : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable(
+                name: "PharmanetTransactionLogTemp");
```

Comment:
This snuck in?

Reply:
Not sure why. I re-generated the migration and it did the same. For now, I removed the code manually for now.